.github/workflows/release-codecov-cli.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow publishes the codecov-cli assets to PyPI, GitHub releases, and
+# our GCS bucket when a release is created.
+name: Build and publish codecov-cli
+
+on:
+  release:
+    types:
+      - created
+
+permissions:
+  contents: read
+
+jobs:
+  # publish_to_pypi:
```

Comment:
disabled until I'm happy with release flow and Github releases assets. Harder to go back once we've pushed to pypi/gcs